debezium-core/src/main/java/io/debezium/heartbeat/Heartbeat.java

```diff
@@ -28,15 +28,15 @@
  */
 public interface Heartbeat {
 
-    public static final String HEARTBEAT_INTERVAL_PROPERTY_NAME = "heartbeat.interval.ms";
+    String HEARTBEAT_INTERVAL_PROPERTY_NAME = "heartbeat.interval.ms";
```

Comment:
This and other lines in that file are unrelated, just housekeeping (but depending on the previous question I need to edit that file anyway).
Everything on an interface is `public static` per default and `final` as well.